corehq/apps/app_manager/static/app_manager/js/app_manager.js

```diff
@@ -247,7 +248,34 @@ hqDefine('app_manager/js/app_manager', function () {
                 $(this).closest('.js-sorted-li').removeClass('appnav-highlight');
             });
         }
-
+        function nestChildModules() {
+            var modulesByUid = {},
+                childModules = [];
+            $(".module").each(function (index, element) {
+                $(element).data('index', index);
+                modulesByUid[ $(element).data('uid') ] = element;
+                if ( $(element).data('rootmoduleid') ) {
+                    childModules.push(element);
+                }
+            });
+            _.each(childModules, function (childModule) {
+                var parent = modulesByUid[$(childModule).data('rootmoduleid')];
+                if (!parent) {
+                    // This child module is orphaned, throw it at the end
+                    $("ul.appnav-module").append(childModule);
+                } else {
+                    addChildModuleToParent(childModule, parent);
+                }
+            });
+        }
+        function addChildModuleToParent(childModule, parent) {
+            var childList = $(parent).find("ul.child-modules");
+            if ( childList.length === 0 ) {
```

Comment:
Nitpick: Are you attached to the style of a space after opening paren/bracket and before closing paren/bracket? I feel like we don't use it much, at least in js.

Reply:
Nevermind, looks like stickler has the same complaint.